test/unit/event.js

```diff
@@ -2876,7 +2867,7 @@ QUnit[ jQuery.fn.click ? "test" : "skip" ]( "Event aliases", function( assert )
 	} );
 } );
 
-// Support: IE9 (remove when IE9 is no longer supported)
+// Support: IE9 only
```

Comment:
We need this standard as we need to know what to use so TBH I'd like to document it before we have a JSCS plugin etc. We can always update the description if during the development of the JSCS plugin we discover the spec is incomplete/inconsistent but IMO it'd be better to have at least such an agreed description than none.